### PR TITLE
chore: use bytes.Equal instead bytes.Compare

### PR DESCRIPTION
**Pull request by testwill**

### CODE REVIEW

File: apps/cloud/handshake.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ func (ch *Handshake) handleV1AuthReply(buffer []byte, handshake *handshakeDetail
 	}
 
 	digest := GenDigest(handshake.hash, []byte(message.Node), []byte(ch.options.Cluster), handshake.cookieHash)
-	if bytes.Compare(message.Digest, digest) != 0 {
+	if !bytes.Equal(message.Digest, digest) {
```
Comment (halturin) [inline apps/cloud/handshake.go:244]: I prefer `== false` instead of `!`. Its more readable.

File: node/network.go [hunk 2]
```diff
@@ -657,7 +657,7 @@ func (n *network) RouteProxyConnectRequest(from ConnectionInterface, request Pro
 		}
 	}
 	checkDigest := generateProxyDigest(request.Creation, cookie, request.PublicKey)
-	if bytes.Equal(request.Digest, checkDigest) == false {
+	if !bytes.Equal(request.Digest, checkDigest) {
```
Comment (halturin) [inline node/network.go:660]: why did you change this? :)

Comment (testwill) [reply node/network.go:660]: more readable and performance

**Comment by halturin** [comment]
not valid anymore
